python/tests/test_aac/plugins/root_schema_must_have_name/test_root_schema_must_have_name.py

```diff
@@ -0,0 +1,52 @@
+from unittest import TestCase
+from typing import Tuple
+from click.testing import CliRunner
+from aac.execute.command_line import cli, initialize_cli
+from aac.execute.aac_execution_result import ExecutionStatus
+from aac.context.language_context import LanguageContext
+
+
+from aac.plugins.root_schema_must_have_name.root_schema_must_have_name_impl import (
+    plugin_name,
+)
+
+
+from aac.plugins.root_schema_must_have_name.root_schema_must_have_name_impl import (
+    root_schema_has_name,
+)
+
+
+class TestRootschemamusthavename(TestCase):
```

Comment:
Hum...this may be part of the auto-generated code.  Need to see if there's something that manages case incorrectly in code generation (assuming that's the source).

Reply:
I manually updated the test class name, but I think this is autogenerated.  We should take a look at that and see if we can update the generator to give a better title case test class name.